### DISCUSSION TITLE
How to setup RTL with webpack

### DISCUSSION THREAD

**Discussion by Seb33300** [twbs/bootstrap#33762] [General]
Hello,

I am working on a multilingual website that require to support LTR, but also RTL for arabic language.

I followed [this documentation](https://getbootstrap.com/docs/5.0/getting-started/webpack/) to setup my project to use bootstrap SCSS with webpack and the ability to override bootstrap variables.

This is working properly when rendering LTR pages.
But when rendering RTL pages, only the grid system is "mirrored".
`start` and `end` keywords used on CSS classes seems to not take effect as expected. (For example, `text-start` still display texts aligned left even on RTL pages)

Maybe this is because I need to generate 2 builds of bootstrap? 1 for LTR and 1 for RTL?
But I cannot find how to do this? Is there any variable I need to set to build bootstrap with RTL? Or something I have to setup in my webpack config?

Or even better, can I generate a build containing both LTR and RTL in the same file?

Thank you

**Comment by ffoodd** [comment]
You need to run RTLCSS on your generated, unminified CSS build. This is explained in [our RTL documentation page](https://getbootstrap.com/docs/5.0/getting-started/rtl/#approach).

Not sure what it implies when using Webpack though, I suggest you search after RTLCSS docs or resources.

**Comment by motahharm** [comment]
Hi @Seb33300 

install RTLcss with npm and use it with web pack

you can read docs at <a href="https://rtlcss.com/learn/">here</a>
